ui/components/app/collectibles-items/collectibles-items.component.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box';
+import Button from '../../ui/button';
+import Typography from '../../ui/typography/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+  TEXT_ALIGN,
+  JUSTIFY_CONTENT,
+  FLEX_DIRECTION,
+  ALIGN_ITEMS,
+  DISPLAY,
+  BLOCK_SIZES,
+  SIZES,
+  FLEX_WRAP,
+} from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
+import { ENVIRONMENT_TYPE_POPUP } from '../../../../shared/constants/app';
```

Comment:
Will be nice to have imports well arranged.

Reply:
how do you prefer the arrangement of imports?